Fix #17335: plugin docs now show correct type for PixelData 'data' attribute (#17336)
OpenRCT2/OpenRCT2: distribution/changelog.txt

- Fix: [#17292] Rows in shortcut key list stay highlighted when cursor leaves list.
- Fix: [#17295] Pause status not cleared when loading a scenario made from a converted paused save.
- Fix: [#17310] Reversed reversible vehicles not imported properly when loading RCT1 parks.

> - Fix: [#17335] [Plugin] Documentation now has the correct type for PixelData 'data' attribute.

Reviewer: Describe the bug, not the fix.
```suggestion
- Fix: [#17335] [Plugin] Documentation has an incorrect type for PixelData 'data' attribute.
```
